lib/galaxy/tools/parameters/basic.py

```diff
@@ -1047,6 +1047,15 @@ def to_dict(self, trans, other_values=None):
         d['textable'] = is_runtime_context(trans, other_values)
         return d
 
+    def validate(self, value, trans=None):
+        if value:
+            if self.multiple:
+                if not isinstance(value, list):
+                    value = [value]
+            else:
+                value = [value]
+            for v in value:
+                super().validate(v, trans)
```

Comment:
```suggestion
        if not value:
            super().validate(value, trans)
        if self.multiple:
            if not isinstance(value, list):
                value = [value]
        else:
            value = [value]
        for v in value:
            super().validate(v, trans)

```
That should fix the no options validator (and the linting)